pkg/apparmor/apparmor_linux.go

```diff
@@ -25,11 +25,10 @@ import (
 // profileDirectory is the file store for apparmor profiles and macros.
 var profileDirectory = "/etc/apparmor.d"
 
-// IsEnabled returns true if AppArmor is enabled on the host. It also checks
-// for the existence of the `apparmor_parser` binary, which will be required to
-// apply profiles.
+// IsEnabled returns true if `apparmor_parser` binary is found on the host,
+// which will be required to apply profiles.
 func IsEnabled() bool {
```

Comment:
I don't think we can just change the meaning of IsEnabled(), this will break anyone who depends on the current meaning.

Reply:
I've added a new IsAppArmorAvailable() and will use that to call from within CRI-O [apparmor config](https://github.com/cri-o/cri-o/blob/8d5c8a7747261804e5ac7a7ed11f5d30f2516e51/internal/config/apparmor/apparmor_linux.go#L26).